src/ChainSafe.GamingSdk.EVM.Unity/build.bat

```diff
@@ -1 +1,5 @@
-dotnet build /property:Unity=true
\ No newline at end of file
+dotnet build /property:Unity=true
+
+pushd ..\..
+dotnet restore
```

Comment:
Shouldn't this happen before the build, since this had to run to fix the errors in the build?

Reply:
No, this serves to fix the state of the solution *after* the build, so you won't be getting the Nethereum errors you've been getting.

Reply:
Do we also need this in the process `dotnet build /property:Unity=True` ?  Or should that be a footnote and used manually based on the return to the editor?

Reply:
Yes, that's what's actually doing the build.

Reply:
What is the purpose of this file?